viewer3d/presets.py

```diff
@@ -121,3 +122,214 @@ def templatePreset(packed_and_poses_and_pdbs=None, *args, **kwargs):
     # Add custom Viewer commands here
 
     return view()
+
+
+@requires_init
+def makeBundle(
```

Comment:
@ajasja Please take a look at this preset, and let me know if the API works generally for users defining their own widgets and update functions. In general, the user can pass a `VBox` or `HBox` widget containing all of their widgets into the viewer to be displayed using `view.modules = VBox([..., ..., ...])`

Reply:
Sounds great! The `makeBundle` preset now just returns a viewer instance for the user to call `view.show()` on. The layer colors have also been updated and the preset takes the `window_size` argument.

Reply:
There is also a `view.set_widgets()` method instead of setting the class instance attribute (which is renamed `view.widgets` instead of `view.widget`) that can take a `Widget` object or list of `Widget` objects .